src/cplscheme/tests/ParallelImplicitCouplingSchemeTest.cpp

```diff
@@ -559,7 +559,7 @@ BOOST_AUTO_TEST_CASE(FirstOrderWithInitializationAndAcceleration)
   const double timeWindowSize     = 0.1;
   const int    maxIterations      = 3;
   const int    extrapolationOrder = 1;
-  const double timestepLength     = timeWindowSize;
+  const double timeStepSize       = timeWindowSize;
```

Comment:
This is maybe an alternative approach to make window vs. step in tests more explicit, if there is no subcycling.

Reply:
Yes. In an ideal world, a comment would be there
```cpp
// solver is not subcycling
const double timeStepSize       = timeWindowSize;
```
I guess, no need to add one now everywhere

Reply:
Added it at some places in 09bdd761e72774a0d440cc0fef0b2671fcaca835, if I came across.